### PR TITLE
remove autofil lnurl-withdraw

### PR DESCRIPTION
**Pull request by ubbabeck**
# About pr

This pr refers to #1057 where the amount of lnurlwithdraw where filled by default. 
## What is done 
- remove the autofilling
- add  _isFixedAmount_  to class  _PayFetchResponse_ and _WithdrawFetchResponse_
- display invoice limit if min and max withdraw are not the equal

### Testing
Test where done with https://lnurl.fiatjaf.com/

### CODE REVIEW

**Comment by tlindi** [comment]
I like this 🙏

File: lib/routes/create_invoice/create_invoice_page.dart [hunk 1]
```diff
@@ -467,11 +482,9 @@ class CreateInvoicePageState extends State<CreateInvoicePage> {
   ) {
     _withdrawFetchResponse = response;
     _descriptionController.text = response.defaultDescription;
-    _amountController.text = account.currency.format(
-      response.maxAmount,
-      includeDisplayName: false,
-      userInput: true,
-    );
+    if (response.isFixedAmount) {
+      _amountController.text = "${response.minAmount}";
```
Comment (erdemyerebasmaz) [inline lib/routes/create_invoice/create_invoice_page.dart:486]: Why are we not using the same format here?

File: lib/routes/lnurl_fetch_invoice_page.dart [hunk 2]
```diff
@@ -218,17 +218,19 @@ class LNURLFetchInvoicePageState extends State<LNURLFetchInvoicePage> {
...
+                          child: Text(
+                            texts.lnurl_fetch_invoice_limit(
+                              acc.currency.format(response.minAmount),
+                              acc.currency.format(response.maxAmount),
+                            ),
+                            textAlign: TextAlign.left,
+                            style: theme.FieldTextStyle.labelStyle,
+                          ),
+                        )
+                      : Padding(padding: const EdgeInsets.only(top: 8)),
```
Comment (erdemyerebasmaz) [inline lib/routes/lnurl_fetch_invoice_page.dart:233]: Is the padding necessary when the amount is fixed? If it's not necessary, I suggest to add this widget to the widget tree as such:
```suggestion
                    if (!response.isFixedAmount) ...[
                      Padding(
                           padding: const EdgeInsets.only(top: 8),
                           child: Text(
                             texts.lnurl_fetch_invoice_limit(
                               acc.currency.format(response.minAmount),
                               acc.currency.format(response.maxAmount),
                             ),
                             textAlign: TextAlign.left,
                             style: theme.FieldTextStyle.labelStyle,
                           ),
                         )
                    ]
```

File: lib/routes/create_invoice/create_invoice_page.dart [hunk 3]
```diff
@@ -209,6 +209,8 @@ class CreateInvoicePageState extends State<CreateInvoicePage> {
                 return acc.validateIncomingPayment(amount);
               }
 
+              final response = widget.lnurlWithdraw;
```
Comment (erdemyerebasmaz) [inline lib/routes/create_invoice/create_invoice_page.dart:212]: It looks like you can use _withdrawFetchResponse variable here directly as this value is assigned to it initially.

**Review by erdemyerebasmaz — APPROVED** [review]

**Pull request merged**